install/windows-10.md

```diff
@@ -343,12 +343,12 @@ mkdir -p ~/.local
 Dieser Befehl erstellt einen Ordner mit dem Namen `.local`, falls dieser nicht schon existiert
 und tut gar nichts, falls dieser Ordner schon existiert.
 
-### <a id="Mambaforge"></a>Mambaforge: Python Installation
+### <a id="Miniforge3"></a>Miniforge3: Python Installation
```

Comment:
We should keep this consistent: Either "Miniforge3: Python Installation" (as in Win10/11) or "Python Installation: Miniforge3" (as in Linux/MacOS)

Reply:
I did a simple search and replace. So if it isn't consistent now, it wasn't before...

Reply:
Yeah, I just thought we could kill two birds with one stone while we're at it. Other than that it's perfectly fine imo

Reply:
done